docs/quickstart.md

```diff
@@ -0,0 +1,428 @@
+* [1] petstore.yaml:14:7 (the error is somewhere at line 14 near the 7th symbol)
+* Operation object should contain `operationId` field (there should be a particular field here)
+* Warning was generated by the operation-operationId rule (a rule that detected the warning)
+
+You got that error because you've changed a property that is strictly defined in the specification. The linter has the [`spec`](./built-in-rules.md#spec) built-in rule that generates an error each time when it finds something that is not defined in the specification.
+
+The warning is a consequence of the error - when you changed the `operationId`, you also changed the object that this property is a part of - the [Operation object](https://spec.openapis.org/oas/latest.html#operation-object). Since the `operationId` is no longer there, you get a warning triggered by the corresponding built-in rule - [operation-operationId](./built-in-rules.md#operation-operationid).
+
+Let's make you definition valid again. If you look at the defined Field Names of the [Operation object](https://spec.openapis.org/oas/latest.html#operation-object) (after the warning description), you can quickly detect what's wrong with your definition file - there cannot be a Field Name with the `operationIdentifier` name, it is invalid. So, change it back to the `operationId` and you're done. In this way, you will fix both the warning and the error because the unexpected property `operationIdentifier` is gone.
+
+To check that everything is correct, run the `lint` command again:
+
+```bash
+$ openapi lint petstore.yaml
+No configurations were defined in extends -- using built in recommended configuration by default.
+
+validating petstore.yaml...
+petstore.yaml: validated in 47ms
+
+Woohoo! Your OpenAPI definition is valid. 🎉
+```
+
+Congratulations! You did a great job and now your OpenAPI definition is valid again!
+
+### split - divide your large definition into smaller constituent parts
+
+With this command, your OpenAPI definition file will be divided into constituent parts to follow the multi-file approach.
+
+In your terminal, type the following:
+
+```bash
+$ openapi split petstore.yaml --outDir openapi
+```
+
+You should get the following response:
+
+```bash
+$ openapi split petstore.yaml --outDir openapi
+Document: petstore.yaml is successfully split
+ and all related files are saved to the directory: openapi 
+
+petstore.yaml: split processed in 44ms
+```
+
+Cool, split completed. But what actually happened?
+
+Open the `openapi` directory. See? That's where the magic took place:
+
+```bash
+.
+├── components
+│   └── schemas
+│       ├── Error.yaml
+│       ├── Pet.yaml
+│       └── Pets.yaml
+├── paths
+│   ├── pets.yaml
+│   └── pets@{petId}.yaml
+└── openapi.yaml
+```
+
+With the `split` command, you can easily switch to a multi-file structure in a blink of an eye even if your definition file became extremely lengthy.
+
+### bundle - combine several definition files into a single one
```

Comment:
This isn't really combining several definition files -- that's what `join` does. In this case it is resolving the reference objects ($ref's) that exist in separate files and pulls them into a single bundled file. I would say something like it:

> bundle - resolves $ref's pointing to other files and pulls them into a single definition file

Reply:
Yeah, my bad, I didn't describe it properly.

However, for me, the text of the summary looks like we are gathering all the $ref's in one definition file.

Probably, my version is a bit clearer:

> bundle - pull separated parts of the OpenAPI definition into a single file

Reply:
Yes. Good job.